src/ui/page/share/dataset/steps/dataSelection/dataSelection.tsx

```diff
@@ -63,24 +63,26 @@ export const DataSelection: FC = () => {
     [handleFileError, removeFile]
   )
 
-  const items: Item[] = files()().map(({ id, name }) => ({
-    content: <p>{name}</p>,
-    id,
-    key: id,
-    leftElement: (
-      <div className="okp4-dataverse-portal-share-dataset-page-file-icon">
-        <Icon name="folder-outlined" />
-      </div>
-    ),
-    rightElement: (
-      <div
-        className="okp4-dataverse-portal-share-dataset-page-file-deletion"
-        onClick={handleFileDeletion(id)}
-      >
-        <Icon name="bin" />
-      </div>
-    )
-  }))
+  const items: Item[] = files()()
+    .map(({ id, name }) => ({
+      content: <p>{name}</p>,
+      id,
+      key: id,
+      leftElement: (
+        <div className="okp4-dataverse-portal-share-dataset-page-file-icon">
+          <Icon name="folder-outlined" />
+        </div>
+      ),
+      rightElement: (
+        <div
+          className="okp4-dataverse-portal-share-dataset-page-file-deletion"
+          onClick={handleFileDeletion(id)}
+        >
+          <Icon name="bin" />
+        </div>
+      )
+    }))
+    .reverse()
```

Comment:
If these changes are purely visual, we might prefer to change list order using CSS.
If the purpose of reversing the list is only for visual presentation, we might consider changing the order using CSS (e.g. using `flex-direction: column-reverse` for flex containers or appropriate grid properties for grid containers). This would be more efficient, especially for larger lists.

Reply:
@0xLodz That's a good idea too! 
We could add an `inverted` property to the list and manage the direction according to the presence or absence of this prop.